proto/grp.proto

```diff
@@ -29,14 +32,20 @@ message Group {
 }
 
 message JoinGroupRequest {
+  // TODO: Add request_context
```

Comment:
haha now in your followup you can just sed -i s#// TODO: Add request_context#context.RequestContext request_context = 1;/g i guess?

anyway thanks again for cleaning these up!

next time you don't have to worry about the TODOs (they just make the CL kind of confusing) -- if you say you'll do it I trust you will :)